src/main/java/org/folio/fqm/repository/EntityTypeRepository.java

```diff
@@ -31,7 +35,10 @@
 public class EntityTypeRepository {
   public static final String ID_FIELD_NAME = "id";
   public static final String TABLE_NAME = "entity_type_definition";
-
+  public static final String REQUIRED_FIELD_NAME = "jsonb ->> 'name'";
+  public static final String REF_ID = "jsonb ->> 'refId'";
+  public static final String WHERE_CLAUSE = "jsonb ->> 'type'";
```

Comment:
This name seems misleading, since it's only a part of the where clause. I think I'd just call it "TYPE_FIELD" or something